Scripts/parameters/assignment.py

```diff
@@ -304,99 +304,69 @@
 # TODO: Trucks and vans
 volume_factors = {
     "car": {
-        "aht": 1. / 0.465,
-        "pt": 1. / 0.094,
-        "iht": 1. / 0.369,
+        "aht": 1. / 0.4179,
+        "pt": 1. / 0.1265,
+        "iht": 1. / 0.3854,
     },
     "car_work": {
-        "aht": 1. / 0.456,
-        "pt": 1. / 0.102,
-        "iht": 1. / 0.433,
+        "aht": 1. / 0.4917,
+        "pt": 1. / 0.2264,
+        "iht": 1. / 0.4806,
     },
     "car_leisure": {
-        "aht": 1. / 0.488,
-        "pt": 1. / 0.089,
-        "iht": 1. / 0.289,
+        "aht": 1. / 0.5597,
+        "pt": 1. / 0.1209,
+        "iht": 1. / 0.3573,
     },
     "transit": {
-        "aht": 1. / 0.478,
-        "pt": 1. / 0.109,
-        "iht": 1. / 0.405,
+        "aht": 1. / 0.4961,
+        "pt": 1. / 0.1607,
+        "iht": 1. / 0.4886,
     },
     "transit_work": {
-        "aht": 1. / 0.445,
-        "pt": 1. / 0.103,
-        "iht": 1. / 0.414,
+        "aht": 1. / 0.709,
+        "pt": 1. / 0.3201,
+        "iht": 1. / 0.4886,
     },
     "transit_leisure": {
-        "aht": 1. / 0.571,
-        "pt": 1. / 0.117,
-        "iht": 1. / 0.373,
-    },
-    "car_first_mile": {
-        "aht": 1. / 0.478,
-        "pt": 1. / 0.109,
-        "iht": 1. / 0.405,
-    },
-    "car_last_mile": {
-        "aht": 1. / 0.478,
-        "pt": 1. / 0.109,
-        "iht": 1. / 0.405,
-    },
-    "train": {
-        "aht": 1. / 0.478,
-        "pt": 1. / 0.109,
-        "iht": 1. / 0.405,
-    },
-    "long_d_bus": {
-        "aht": 1. / 0.478,
-        "pt": 1. / 0.109,
-        "iht": 1. / 0.405,
-    },
-    "airplane": {
-        "aht": 1. / 0.478,
-        "pt": 1. / 0.109,
-        "iht": 1. / 0.405,
```

Comment:
I think it would be good to keep these after all, because they will be assigned on the hour-level in the regional sub-models and the model user may be interested in summing all link flows on day level.